discovery/README.md

```diff
@@ -10,8 +10,19 @@ Docker Swarm comes with multiple Discovery backends.
 
 ## Backends
 
+You use a hosted discovery service with Docker Swarm. The service
+maintains a list of IPs in your swam. There are several available
+services, such as `etcd`, `consul` and `zookeeper` depending on what
```

Comment:
sgtm; just adding possible alternatives because of https://github.com/docker/swarm/pull/1178#issuecomment-136859981 :smile: